app/models/merchant.rb

```diff
@@ -1,6 +1,7 @@
 class Merchant <ApplicationRecord
   has_many :items, dependent: :destroy
   has_many :item_orders, through: :items
+  has_many :users, -> {where role: :merchant}
```

Comment:
Nice job on linking the merchant role to the user optionally.

Reply:
I did not know we could use this syntax. Awesome!

Reply:
I was also unaware that you can do that!